src/runner.ts

```diff
@@ -408,6 +412,22 @@ export class ResolveRunner implements Types.IResolveRunner {
       if (uriResolver) {
         lookupResult.resolved = await uriResolver.resolve(Utils.uriToJSONPointer(ref));
 
+        // support custom transformers
+        if (this.transformDereferenceResult) {
```

Comment:
Probably better in the resolve function? https://github.com/stoplightio/json-ref-resolver/blob/master/src/runner.ts#L235

Reply:
Done